@xen-orchestra/lite/src/components/AccountButton.vue

```diff
@@ -1,38 +1,73 @@
 <template>
-  <button class="account-button">
-    <FontAwesomeIcon class="user-icon" :icon="faCircleUser" />
-    <FontAwesomeIcon class="dropdown-icon" :icon="faAngleDown" />
-  </button>
+  <AppMenu placement="bottom-end" shadow>
+    <template #trigger="{ open, isOpen }">
+      <button :class="{ active: isOpen }" class="account-button" @click="open">
```

Comment:
Incorrect type for `@click="open"`.

The `open` function should look like:
```ts
// AppMenu.vue

const open = (event: MouseEvent) => {
  if (isOpen.value) {
    isOpen.value = false;
    return;
  }

  isOpen.value = true;

  nextTick(() => {
    onClickOutside(menu, () => (isOpen.value = false), {
      ignore: [event.currentTarget as HTMLElement],
    });

    placement(event.currentTarget as HTMLElement, unrefElement(menu), {
      placement:
        unref(isParentHorizontal) !== false ? "bottom-start" : "right-start",
    });
  });
  return;
};
```

Reply:
I fixed the `MouseEvent`.

However, the `void` return type does not enforce not returning any value, so the code is valid.

`void` is just a way to say "you can return nothing or anything you want, but I don't care because I will not use this return value"

More details on [TypeScript handbook](https://www.typescriptlang.org/docs/handbook/2/functions.html)